--- a/src/ui/article_content/view.rs	1969-12-31 18:00:01.000000000 -0600
+++ b/src/ui/article_content/view.rs	2026-06-04 22:06:24.509050549 -0500
@@ -475,24 +475,13 @@
 
         let inner_link_url_for_hint = url_for_hint.clone();
         let inner_link_hint_iterator = hint_iterator.clone();
-        let link_alt_text_style = Style::new()
-            .fg(*config.theme.color_palette().accent_primary())
-            .add_modifier(Modifier::UNDERLINED);
+        let link_alt_text_style = Style::new().fg(*config.theme.color_palette().foreground());
         let link_url_text_style = Style::new().fg(*config.theme.color_palette().foreground());
-        let link_hint_style = Style::new()
-            .fg(*config.theme.color_palette().highlight())
-            .add_modifier(Modifier::BOLD);
 
         let inner_image_url_for_hint = url_for_hint.clone();
         let inner_image_hint_iterator = hint_iterator.clone();
-        let image_alt_text_style = Style::new()
-            .fg(*config.theme.color_palette().accent_primary())
-            .add_modifier(Modifier::UNDERLINED);
+        let image_alt_text_style = Style::new().fg(*config.theme.color_palette().foreground());
         let image_url_text_style = Style::new().fg(*config.theme.color_palette().foreground());
-        let image_hint_style = link_hint_style;
-
-        let image_icon = config.icon_set.image_icon();
-        let url_icon = config.icon_set.url_icon();
 
         let text = {
             let renderer = RendererBuilder::new()
@@ -503,10 +492,7 @@
                     url_for_hint
                         .entry(hint.to_owned())
                         .or_insert(url.to_owned());
-                    let mut spans = vec![
-                        Span::styled(format!("{hint}{url_icon}"), link_hint_style),
-                        Span::styled(alt.to_owned(), link_alt_text_style),
-                    ];
+                    let mut spans = vec![Span::styled(alt.to_owned(), link_alt_text_style)];
                     if show_url {
                         spans.push(Span::styled(format!("({url})"), link_url_text_style));
                     }
@@ -519,10 +505,7 @@
                     url_for_hint
                         .entry(hint.to_owned())
                         .or_insert(url.to_owned());
-                    let mut spans = vec![
-                        Span::styled(format!("{hint}{image_icon}"), image_hint_style),
-                        Span::styled(alt.to_owned(), image_alt_text_style),
-                    ];
+                    let mut spans = vec![Span::styled(alt.to_owned(), image_alt_text_style)];
 
                     if show_url {
                         spans.push(Span::styled(format!("({url})"), image_url_text_style));
